setup.py

```diff
@@ -98,6 +98,7 @@ def build_extension(self, ext):
         cmake_args += [f'-DMAX_DET_ORB={self.max_det_orb}']
         cmake_args += [f'-DENABLE_CODECOV={str(self.enable_codecov).upper()}']
         cmake_args += [f'-DENABLE_ForteTests=TRUE']
+        cmake_args += ['-DCMAKE_OSX_SYSROOT=' + os.environ["CONDA_BUILD_SYSROOT"]]
```

Comment:
This assumes the user is using conda?

Reply:
I wonder if we should make this optional.

Reply:
Good call - I need this line locally to build Psi and Forte. I'll make this into a comment instead.